plugins/lang-javascript/src/main/java/org/elasticsearch/script/javascript/JavaScriptScriptEngineService.java

```diff
@@ -138,6 +139,8 @@ public static void init() {}
     public JavaScriptScriptEngineService(Settings settings) {
         super(settings);
 
+        deprecationLogger.deprecated("Javascript scripts are deprecated.  Use Painless scripts instead.");
```

Comment:
Kind of unusual to put the two spaces `  ` after the period. Good grammar though. :)

More seriously, I wonder if we should be showing the language name as far as scripts are concerned?

```
[javascript] scripts are deprecated. use [painless] scripts instead.
```

It gives a little more emphasis and it highlights what they need to do to fix their issue (easier said than done from JS or Python in particular).